lib/interferon/version.rb

```diff
@@ -1,3 +1,3 @@
 module Interferon
-  VERSION = "0.0.8"
+  VERSION = "0.0.8.1"
```

Comment:
so, this breaks semver, which is not recommended.

i recommend avoiding version bumps with each of your PRs. not every change demands a new version, and i don't think every change should be published to a gem server. if you want to do local testing, you can point your local bundle config to a directory of this gem, or to the repository. then we can release new versions when they've actually been tested.